views/form.ejs

```diff
@@ -0,0 +1,14 @@
+<form role="form" action="<%= base %>/save" method="POST" class="form-horizontal form-bordered" id="validateForm">
+    <input type="hidden" id="id" name="id" value="<%= inst.id %>" />
+    <div class="form-body">
+      <% attributes.forEach(function(attr) { %>
+        <% attr.value = inst[attr.name] %>
+        <%- include("inputs/"+attr.type+".ejs", attr) %>
```

Comment:
This'll error if a type is missing, right? (JSON, Array types) That's probably fine now, will be better when these includes are via renderPartial() instead and can be overridden in part.